quiz-template.html

```diff
@@ -1,148 +1,188 @@
-<html>
+<!DOCTYPE html>

```

Comment:
`<!DOCTYPE html>` is required to tell the browser that the document is HTML5 so that it won't go into quirks mode.